saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -308,3 +312,245 @@ def perform_mutation(cls, _root, info, **data):
+        error_type_class = OrderError
+        error_type_field = "order_errors"
+
+    @classmethod
+    def clean_input(cls, info, order_id, input):
+        amount_to_refund = input.get("amount_to_refund")
+        include_shipping_costs = input.get("include_shipping_costs")
+        if amount_to_refund is not None and include_shipping_costs:
+            raise ValidationError(
+                {
+                    "amount_to_refund": ValidationError(
+                        (
+                            "The amountToRefund and includeShippingCosts are mutually "
+                            "exclusive and attempts to call mutation with both options "
+                            "is incorrect."
+                        ),
+                        code=OrderErrorCode.INVALID.value,
+                    ),
+                    "include_shipping_costs": ValidationError(
+                        (
+                            "The amountToRefund and includeShippingCosts are mutually "
+                            "exclusive and attempts to call mutation with both options "
+                            "is incorrect."
+                        ),
+                        code=OrderErrorCode.INVALID.value,
+                    ),
+                }
+            )
+        qs = order_models.Order.objects.prefetch_related("payments")
+        order = cls.get_node_or_error(
+            info, order_id, field="order", only_type=Order, qs=qs
+        )
+        payment = order.get_last_payment()
+        if not payment or not payment.can_refund():
+            raise ValidationError(
+                {
+                    "order": ValidationError(
+                        "Order cannot be refunded.",
+                        code=OrderErrorCode.CANNOT_REFUND.value,
+                    )
+                }
+            )
+
+        if amount_to_refund is not None and amount_to_refund > payment.captured_amount:
+            raise ValidationError(
+                {
+                    "amount_to_refund": ValidationError(
+                        (
+                            "The amountToRefund is greater than the maximal possible "
+                            "amount to refund."
+                        ),
+                        code=OrderErrorCode.CANNOT_REFUND.value,
+                    ),
+                }
+            )
+        cleaned_input = {
+            "amount_to_refund": amount_to_refund,
+            "include_shipping_costs": include_shipping_costs,
+            "order": order,
+            "payment": payment,
+        }
+        order_lines_data = input.get("order_lines")
+        fulfillment_lines_data = input.get("fulfillment_lines")
+
+        if not order_lines_data and not fulfillment_lines_data:
+            raise ValidationError(
+                {
+                    "order_lines": ValidationError(
+                        "The orderLines or fulfillmentLines cannot be empty.",
```

Comment:
The error code should be:
```suggestion
                        "The orderLines and fulfillmentLines cannot be empty.",
```
As you have `and` between checks.